cypress/integration/data-import/match-on-pol-and-update-items.spec.js

```diff
@@ -259,13 +270,12 @@ describe('ui-data-import: Match on POL and update related Instance, Holdings, It
         materialTypeId
       ));
     Orders.checkIsOrderCreated(firstItem.orderNumber);
-
     // open the first PO
     openOrder(firstItem.orderNumber);
     OrderView.checkIsOrderOpened('Open');
     OrderView.checkIsItemsInInventoryCreated(firstItem.title, 'Main Library');
```

Comment:
Are you sure that 'Main Library' exists?

Reply:
yes, I got it 'cy.getLocations({ limit:1, query:'name="Main Library"' })'

Reply:
I mean that in environment this location can be not exists. If you need this location you need to create it

Reply:
I created location